units/patch_intent.go

```diff
@@ -361,6 +361,13 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 		}
 		if patchDoc.IsParent() {
 			for _, childPatch := range patchDoc.Triggers.ChildPatches {
+				ghSub := event.NewGithubStatusAPISubscriber(event.GithubPullRequestSubscriber{
```

Comment:
I think you can do ghSub.ChildId = childPatch

Reply:
yeah, I probably can. I'll try.

Reply:
I can't access ghSub without casting the type. I'm not sure it's worth it...

Reply:
getting panics. reverting to the original.

Reply:
what was the cause of the panics?

Reply:
I think the panic was caused by trying to access `target.ChildId`